package.json

```diff
@@ -56,14 +139,14 @@
     "karma-phantomjs-launcher": "^1.0.4",
     "less": "2.6.1",
     "load-grunt-tasks": "0.4.0",
-    "lodash": "3.10.1",
     "protractor": "1.7.0",
     "protractor-screenshot-reporter": "^0.0.5",
     "serve-static": "1.10.2",
     "time-grunt": "0.3.2"
   },
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=4.0.0",
+    "yarn": ">= 1.0.0"
   },
   "scripts": {
```

Comment:
As far as you can tell from this change, any need to update the `scripts` block with new commands? Or is everything basically the same from the outside.

Reply:
Everything should be the same from the outside. I adjusted the scripts and grunt tasks so that it should be seamless.